sections/Deals/AcceptOrReject/AcceptOrRejectSection/AcceptOrRejectSection.tsx

```diff
@@ -186,11 +197,19 @@ const AcceptOrRejectDeal: FC<AcceptOrRejectDealProps> = ({
 					</>
 				),
 				subText: (
-					<TokenDisplay
-						symbol={underlyingDealTokenSymbol}
-						address={deal?.underlyingDealToken}
-						displayAddress={true}
-					/>
+					<FlexDivCol>
+						<StyledFlexDiv>
```

Comment:
last thing for this PR is we need a Tooltip explaining that the total supply is not always accurate especially for pre tokens that may only represent a portion of the official total supply when the token is launched

Reply:
I will start syncing the official subgraph now so you can put back the old subgraphs as well